.github/workflows/tpu_ci.yml

```diff
@@ -33,7 +31,6 @@ jobs:
           PJRT_DEVICE: TPU
         # Jax is needed for pallas tests.
         run: |
-          pip install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu --user
```

Comment:
Are these changes intentional?

Reply:
Thanks for catching it, I need to undo the change to GHA

Reply:
Done recover to HEAD